Source/WebCore/page/scrolling/ScrollAnchoringController.cpp

```diff
@@ -369,7 +369,8 @@ void ScrollAnchoringController::adjustScrollPositionForAnchoring()
         }
 #endif
         auto newScrollPosition = m_owningScrollableArea.scrollPosition() + IntPoint(adjustment.width(), adjustment.height());
-        LOG_WITH_STREAM(ScrollAnchoring, stream << "ScrollAnchoringController::updateScrollPosition() for frame: " << frameView() << " for scroller: " << m_owningScrollableArea << " adjusting from: " << m_owningScrollableArea.scrollPosition() << " to: " << newScrollPosition);
+        ALWAYS_LOG_WITH_STREAM(stream << "ScrollAnchoringController::updateScrollPosition() is main frame: " << frameView().frame().isMainFrame() << " for scroller: " << m_owningScrollableArea << " adjusting from: " << m_owningScrollableArea.scrollPosition() << " to: " << newScrollPosition);
```

Comment:
(It would also be good to double check that `m_owningScrollableArea` doesn't contain any sensitive information, as well)

Reply:
I was considering changing this to log the element of the scrollable area as well as the anchor element. Should we be concerned about logging the element (as it would show generally where the user was looking on the page) or are we just worried about logging the url?

Reply:
I suppose the elements could also have revealing class names as well.

Reply:
Yeah, class and id names (while mostly innocuous) _could_ potentially reveal what websites the user was visiting.

Reply:
Right I'll split those logs into debug logs and keep just the mainframe/main scroller and scroll position logging for release.

Reply:
Sounds good! rs=me